packages/create-modular-react-app/src/index.ts

```diff
@@ -142,7 +142,7 @@ export default async function createModularApp(argv: {
       'prettier',
       'modular-scripts',
       'eslint-config-modular-app',
-      'typescript@>=4.2.1 <4.5.0',
+      'typescript@^4.8.3',
```

Comment:
Remember to add a changelog to bump CMRA as major, since this is breaking.

Reply:
Yep makes sense, done